client/client.go

```diff
@@ -261,19 +265,16 @@ func (c *LogClient) getInclusionProof(ctx context.Context, leafHash []byte, tree
 	return nil
 }
 
-func (c *LogClient) queueLeaf(ctx context.Context, leaf *trillian.LogLeaf) error {
-	// Queue Leaf
-	req := trillian.QueueLeafRequest{
+// QueueLeaf add a leaf to a trillian log without blocking.
+// AlreadyExists is considered a success case by this function.
+func (c *LogClient) QueueLeaf(ctx context.Context, data []byte) error {
```

Comment:
This pull request adds 2 public entrypoints (`QueueLeaf`, `WaitForInclusion`), so the overall description is misleading.  (To put it another way, anyone hunting through commit logs trying to figure out when `QueueLeaf` was added is not going to spot this change.)